.github/ISSUE_TEMPLATE/job_opening.md

```diff
@@ -30,12 +30,3 @@ Write a few sentences describing the position here.
 #### Reports To
 - First Name: [FIRST NAME HERE]
 - Discord Username: [DISCORD USERNAME HERE]
-
```

Comment:
actually the `Reports To` section is currently not used as well right?

Reply:
Yeah, I was thinking that too. But as every team has a team lead, it could go here. 🤔